### PR TITLE
Ajax images

### PR DESCRIPTION
**Pull request by mikehuston**

### CODE REVIEW

File: app/assets/javascripts/custom_order_images.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+    // var popupDiv = $('<div id="itemInfo" style="padding: 2ex; position: absolute; border: 2px double grey; backgroud: wheat;"></div>');
+    // popupDiv.hide().appendTo($('body'));
+    // $('span').click(ImagePopup.getImageInfo);
+    // $('span').mouseenter(ImagePopup.getImageInfo);
+    $('.item').mouseover(ImagePopup.getImageInfo);
+    // $('#closeLink').click(ImagePopup.hideImageInfo);
   }
   ,getImageInfo: function() {
-    console.log('here');
+    console.log("HELLO");
```
Comment (matthewnguyen) [inline app/assets/javascripts/custom_order_images.js]: should we still be displaying this stuff to the console?????

Comment (mikehuston) [inline app/assets/javascripts/custom_order_images.js]: Fixed! Great catch Matt!

**Comment by eshiue328** [comment]
looks good!

**Comment by twiet** [comment]
:+1:

**Pull request merged**